lib/moodle_rb/version.rb

```diff
@@ -1,5 +1,5 @@
 module MoodleRb
-  VERSION = '2.1.5' unless defined?(self::VERSION)
+  VERSION = '2.1.6' unless defined?(self::VERSION)
```

Comment:
I think this warrants a minor version bump - we usually follow [Semantic Versioning](https://semver.org/#spec-item-7)